tests/test_rawpy.py

```diff
@@ -0,0 +1,20 @@
+"""Tests for rawpy plugin
+"""
+
+import os
+import rawpy
+import imageio.v3 as iio
+
+
+def test_nef_local(test_images):
+    """Test for reading .nef file from .test_images dir.
+    """
+    # Check image path
+    im_path = str(test_images / "Nikon.nef")
+    assert os.path.exists(im_path)
+
+    # Test if plugin's content mathces rawpy content
+    im_plugin = iio.imread(im_path)
+    im_rawpy = rawpy.imread(im_path)
+    assert im_plugin.all() == im_rawpy.raw_image.all()
```

Comment:
This will not do what you expect. `ndarray.all()` will return `True` if all elements in the array evaluate to `True` and `False` otherwise. In other words, the current logic checks if both images have at least one pixel with value `0`.

Instead, use the following check:

```python
assert np.allclose(im_plugin, im_rawpy)
```

Sidenote: In my other tests, I tend to use the names
```
actual = iio.imread(im_path)
expected = rawpy.imread(im_path)
```

You are, obviously, free to use any variable name you like, but I think this choice makes it quite clear what the purpose of these arrays is, i.e., it is immediately obvious that `expected` contains the desired result whereas `actual` is what the plugin has produced.

Reply:
Updated the file accordingly. Thanks for the explanation.